tests/tap.sh

```diff
@@ -36,35 +36,27 @@ for ((i=0;i<${#backends[@]}; ++i)); do
     # grep to skip test if backend cannot handle resource
     if grep -F -q -e 'backend cannot use resource' \
             -e 'OCCA backend failed' ${output}.err; then
-        printf "ok $i0 # SKIP $1 $backend\n"
-        printf "ok $i1 # SKIP $1 $backend stdout\n"
-        printf "ok $i2 # SKIP $1 $backend stderr\n"
+        printf "ok $i0 # SKIP - occa mode not supported $1 $backend\n"
+        printf "ok $i1 # SKIP - occa mode not supported $1 $backend stdout\n"
+        printf "ok $i2 # SKIP - occa mode not supported $1 $backend stderr\n"
         continue
     fi
 
-    # grep to skip test if backend cannot handle Blocked Restriction
-    if grep -F -q -e 'Backend does not support ElemRestrictionCreateBlocked' \
-            ${output}.err; then
-        printf "ok $i0 # SKIP $1 $backend\n"
-        printf "ok $i1 # SKIP $1 $backend stdout\n"
-        printf "ok $i2 # SKIP $1 $backend stderr\n"
+     # grep to skip test if backend chooses to whitelist test
+    if grep -F -q -e 'Backend does not implement' \
+            ${output}.err ; then
+        printf "ok $i0 # TODO - not implemented $1 $backend\n"
+        printf "ok $i1 # TODO - not implemented $1 $backend stdout\n"
```

Comment:
[Specification](http://testanything.org/tap-specification.html#todo-tests) says TODO should be marked `not ok`. I don't have a problem with prove's behavior, but I think we should try to follow the standard when using directives. There are other TAP consumers including Jenkins, which we might use to get CI on GPU or other DOE hardware/software stacks.

Reply:
I mixed it up; I thought TODO was another type of conditional pass. Fixed now.